app/scripts/routes.js

```diff
@@ -1010,9 +1010,19 @@
             .when('/selfservice/createuser/:clientId',{
                 templateUrl: 'views/selfservice/createuser.html'
             })
+            .when('/externalservicesCB/CreditBureau', {
+                templateUrl: 'views/administration/CreditBureauSummary.html'
+            })
+             .when('/externalservicesCB/CreditBureau/addcb', {
+                 templateUrl: 'views/administration/addNewCreditBureau.html'
+            })
+            .when('/externalservicesCB/CreditBureau/mapcblp', {
+                templateUrl: 'views/administration/MapCreditBureauToLP.html'
+            })
             .otherwise({
-                templateUrl: "views/errors/404.html"
```

Comment:
is this needed.

Reply:
comment has been addressed and reverted the changes.